corobo/docker-compose.yml

```diff
@@ -4,3 +4,10 @@ services:
     image: meetmangukiya/corobo
     restart: always
     env_file: .secrets
```

Comment:
Please add `environment:` below this, also set the host for answers to be `answers` since it's linked as `answers`.